var/spack/repos/builtin/packages/py-opentuner/package.py

```diff
@@ -17,7 +17,7 @@ class PyOpentuner(PythonPackage):
     # No support for Python 3 yet
     depends_on('python@2.7:2.8', type=('build', 'run'))
 
-    depends_on('py-argparse@1.2.1:', type=('build', 'run'))
+    depends_on('py-argparse@1.2.1:', type=('build', 'run'), when='^python@:2.6')
```

Comment:
I think this one might actually be always required. If the `argparse` module is not installed, `setuptools` will automatically download and install it.

Reply:
`python@2.7:2.8` - Python 2.7 has a built-in `argparse` module, which is the same as offered by this package. If you think this change is required - I will do it.

Reply:
Yes, but since it is listed in `install_requires`, I think `setuptools` will try to automatically install it if the pip module isn't installed. You can try this yourself and see if it happens. Technically, this is a bug in their `setup.py` that should probably be reported to the `opentuner` developers.